modules/jaeger-sidecar-manual.adoc

```diff
@@ -26,11 +26,11 @@ spec:
     spec:
       containers:
         - name: example-app
+          image: acme/myapp:myversion
           ports:
             - containerPort: 8080
               protocol: TCP
         - name: jaeger-agent
-          image: acme/myapp:myversion
```

Comment:
You need to now add `image: registry.redhat.io/distributed-tracing/jaeger-agent-rhel7:<version>` back here.